spec/controllers/automated_tests_controller_spec.rb

```diff
@@ -8,14 +8,43 @@
       # TODO: write tests
     end
     context 'GET manage' do
-      before :each do
```

Comment:
I don't understand why this block is removed and then just repeated a bunch of times below. You can just leave this one in place.

Reply:
For the test case "When the test grouping, membership already exists" I had to create everything before calling the "get :manage" 
If I only one before block for all the test cases that will create the test_grouping, membership already. So I can't test the mentioned test case.

Reply:
in that case please use `let!` variables instead of repeating the before block

Reply:
you know what, never mind, this is fine